tests/tools/oesign/test-enclave/host/host.c

```diff
@@ -49,6 +69,35 @@ int main(int argc, const char* argv[])
         }
     }
 
+    if (_is_kss_supported())
```

Comment:
If this is passing in bors, I suspect that's a side effect of oesign only signing with KSS flag on KSS-enabled build machines, which is a bug. 

I think you're going to restructure this test, because once enclaves can be signed on non-KSS machines with KSS properties, the `oe_create_enclave` on line 45 will fail before it gets to this check, so you're going to have to introduce a mechanism to skip entire tests based on detection of this bit, and a parameter passed into the host to indicate whether a specific test expects KSS or not. 

See tests/debug-mode for an example.

Reply:
Removed _is_kss_supported from oesign to test code

Reply:
After I remove _is_kss_suuported from oesign, test will fail here so I will update the test here like the following flow
    if ((result = oe_create_oesign_test_enclave(
             argv[1], OE_ENCLAVE_TYPE_AUTO, flags, NULL, 0, &enclave)) != OE_OK)
    {
        if (!(result == OE_PLATFORM_ERROR && is_kss_supported() &&  non-zero KSS properties))
            oe_put_err("oe_create_crypto_enclave(): result=%u", result);
        else
        {
            return 0;
        }
        
    }

Reply:
Updated in #3700